openedx/features/wikimedia_features/meta_translations/models.py

```diff
@@ -152,6 +154,13 @@ def apply_all_translations(self):
         for wikitranslation in existing_mappings:
             wikitranslation.applied = True
             wikitranslation.save()
+    
+    def get_parsed_data(self, data_type, data):
+        """
+        Transform raw_data into parsed_data
+        """
+        if data_type == 'content' and self.block_type in TRANSFORMER_CLASS_MAPPING:
```

Comment:
create list of data_types in generic app settings and update check i.e if data_type in settings.DATA_TYPES_WITH_PARCED_KEYS.